scripts/build/release-packages.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+GRAFANA_TAG=${1:-}
+RELEASE_CHANNEL="latest"
+
+if echo "$GRAFANA_TAG" | grep -q "^v"; then
+	_grafana_version=$(echo "${GRAFANA_TAG}" | cut -d "v" -f 2)
+else
+  echo "Provided tag is not a version tag, skipping packages release..."
+	exit
+fi
+
+if grep -q "beta" <<< "$GRAFANA_TAG"; then
+  RELEASE_CHANNEL="next"
+fi
+
+echo "$_grafana_version"
+
+# Get current version from lerna.json
+# Since this happens on tagged branch, the lerna.json version and package.json file SHOULD be updated already
+# as specified in release guideline
+PACKAGE_VERSION=$(grep '"version"' lerna.json | cut -d '"' -f 4)
```

Comment:
This expects `lerna.json` to be in your work dir, but as you `cd` into `scripts/build` before running this script that won't be the case. You could take in the path to lerna.json as an arg or figure out the correct relative path or just run the script from the root of the project.

Reply:
Yeah, you are right!

Reply:
Fixed in 99a8244514af2d7710b38adbceae95a31babe460